convex/slugs.ts

```diff
@@ -0,0 +1,63 @@
+import { mutation, query } from "./_generated/server";
+import { v } from "convex/values";
+
+export const getSlug = query({
+  args: {
+    slug: v.string(),
+  },
+  handler: async (ctx, args) => {
+    const slug = await ctx.db
+      .query("slugs")
+      .filter((s) => s.eq(s.field("slug"), args.slug))
+      .first();
+    console.log("getSlug", slug);
+  },
+});
+
+export const getAllSlugs = query({
+  handler: async (ctx) => {
+    const slugs = await ctx.db.query("slugs").collect();
+    return slugs;
+  },
+});
+
+export const createSlug = mutation({
+  args: {
+    slug: v.string(),
+    // startTime: v.string(),
```

Comment:
This is intentially commented out for a future PR after @20jasper checks in the PR